new-frontiers-14/frontier-station-14
# Honey

<!
<img width="406" alt="honeyPRStuff" src="https://github.com/user-attachments/assets/a49d5d97-7a02-4106-85bf-34e7b2da4b88">
<img width="362" alt="honeyPRStuff2" src="https://github.com/user-attachments/assets/52ce3ea5-e4cb-46e1-b248-3dd81e4071e4">
-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR


I added Honey, Bees in the hydroponics and a few recipes with honey. I also created honey iced tea which is a combo of honey and iced tea.
TO DO:
- [x] Replace seed packet sprite with jar of bees sprite
- [ ] Get a better Honey Iced Tea metamorphic sprite
## Why / Balance

Add more food and variety to cooking system
## How to test

Cook each recipe, grow some bees and turn honeycomb into honey. Make sure honey container is making honey and comb juices into honey.
## Media


## Requirements



## Breaking changes


**Changelog**


:cl:
-add: Honey, Honey recipes, and Bees
